content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -357,6 +359,36 @@ and the Pod's `hostNetwork` field is true. Then `host`, under `httpGet`, should
 to 127.0.0.1. If your pod relies on virtual hosts, which is probably the more common
 case, you should not use `host`, but rather set the `Host` header in `httpHeaders`.
 
+For an HTTP probe, the kubelet sends two additional headers in addition to standard `Host` header:
+`User-Agent` and `Accept`. Default values for these headers are `kube-probe/1.20`
+(where `1.20` is a kubelet version) and `*/*` respectively.
```

Comment:
We could definitely get the static site generator to fill in the actual version number of Kubernetes based on the version of the documentation that the reader is looking at.
```suggestion
`User-Agent` and `Accept`. The default values for these headers are `kube-probe/{{< skew latestVersion >}}`
(where `{{< skew latestVersion >}}` is the version of the kubelet ) and `*/*` respectively.
```